rapier/rules-update-2020-07.md

```diff
@@ -0,0 +1,63 @@
+--- 
+title: Updates from latest Society Rules - July 2020
+layout: default 
+--- 
+ 
+* TOC 
+{:toc} 
+ 
+**Author**: Master Brian de Moray, KRM 
+ 
+**Last Modified**: {{ page.last_modified_at | date: '%Y-%m-%d' }}
+ 
+**Revision**: {{ site.git.last_commit.short_sha }}
+
+# Rubber-tipped Spears
+
+Atlantia now allows for the use of rubber-tipped spears in both rapier and C&T combat.  Rapier Spears are specifically precluded from use in light rapier.  For the foreseeable future, all Rapier Spear activities must be actively marshaled by a rapier marshal with a Rapier Spear authorization.
+
+All those marshaling Rapier Spear activity should record and submit detailed feedback in the Event Report as to the efficacy, safety, and any issues raised during Rapier Spear activities.
+
+The marshal-in-charge of an activity may limit Rapier Spears on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may not use Rapier Spears."
+
+# Parry-Only Daggers
+
+All equipment with a steel or rubber blade must pass the blade requirements.  Equipment without a blade is not an offensive weapon in the context of Rapier in Atlantia.  As an example, an armored combat rattan dagger does not have a blade, and therefore can be used as a parrying device, similar to the already approved wooden dowel rod or rolling pin.
+
+# Knee-Walking
+
+Knee-Walking is allowed by society policy at all times.  The marshal-in-charge of an activity may limit knee-walking on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may only knee walk to engagement."
+
+# C&T Melee
+
+The new society rules allow for melees using C&T combat.  All participants in C&T melee must hold an active C&T authorization and must be actively marshaled by someone approved to marshal C&T combat in Atlantia.  Participants with a Rapier Spear authorization without a C&T authorization may participate in C&T melees using only a Rapier Spear.

```

Comment:
Actively marshaled?

Reply:
Is it safe for non-c&t authorized fighters to participate in this combat?  Would they be required to be armored as C&T fighters?  The authorization process is how we ensure that people are prepared for combat in different circumstances.

Reply:
addressed